src/content/docs/journey-app-slow/root-causes.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Prepare to triage your application
+metaDescription: 
+---
+
+import journeyWebTrans from 'images/journey_screenshot-web-transactions.webp'
+
+Your app is slow. Now what? How do you track down the root cause of your app's slowness?
+
+This tutorial walks you through how to use New Relic to triage your application and identify the root cause of your app's degraded performance. You'll start by "instrumenting" your app with an agent, which means installing a piece of code that reports data about your app to New Relic. Using this data in New Relic, you'll dig into the performance of your application and identify slow transactions, slow database queries, or slow external services as your issue - or possibly all three!
+
+
+
+## Prerequisite: instrument your app [#prereq]
+
+Before you can improve your site's performance, you need the ability to see what's going on. New Relic monitoring always starts with an **integration**, or a piece of code that collects data from a relevant technology and reports it to New Relic. To monitor your application's performance, you will use an agent created specifically for your app's language.
```

Comment:
Above we define an agent as:

> a piece of code that reports data about your app to New Relic

Here we define an integration as:

> a piece of code that collects data from a relevant technology and reports it to New Relic.

This is of course because the NR concepts have a fair amount of overlap. Still, I would use somewhat different and more specific language to help the reader understand how the two concepts interact.